LLReactiveMatchers/Helpers/LLSignalTestRecorder.m

```diff
@@ -33,33 +33,34 @@ @interface LLSignalTestRecorder ()
 @implementation LLSignalTestRecorder
 
 - (id) init {
-    if( (self = [super init]) ) {
-        self.receivedEvents = [NSMutableArray array];
-        self.activeThreadsInReceivedEvents = [NSMutableSet set];
-        self.passthrough = [RACReplaySubject replaySubjectWithCapacity:RACReplaySubjectUnlimitedCapacity];
-        [self startCountingSubscriptions];
-    }
-    return self;
+  if( (self = [super init]) ) {
```

Comment:
Please keep his original 4 spaces convention - there are too many redundant lines that were changed here.